src/components/FullPageWebView/FullPageWebView.tsx

```diff
@@ -196,11 +198,26 @@ const FullPageWebView = ( ) => {
     }, [navigation, params.loggedIn, params.title] )
   );
 
+  const fontScalingJS
+  = `
+    if( ${isLargeFontScale} ){
+      document.body.style.fontSize = '40px';
```

Comment:
Do you really want to hard-code the size like this? Did you try a percentage?

Reply:
I'll try percentages